classes/game.rb

```diff
@@ -0,0 +1,17 @@
+class Game
```

Comment:
I think you need to make Game inherit from Item class like this:
`class Game < Item`

Also don't forget to require item.rb file in the beginning of the file:
`require './classes/item'` or `require_relative 'item'`

Please note this UML for reference:
![image](https://github.com/Zilola-Nazarova/ruby-capstone-project/assets/61951420/303b2034-7133-4701-9b75-549176309ffc)
As you can see Game class is a child of the Item class.